src/index.tsx

```diff
@@ -30,26 +31,28 @@ const queryClient = new QueryClient();
 // MEMO: temporarily removed React.StrictMode. We should add back when react-spectrum handles
 // it across their library. (Issue: https://github.com/adobe/react-spectrum/issues/779#issuecomment-1353734729)
 ReactDOM.render(
-  <Router>
-    <QueryClientProvider client={queryClient}>
-      <HelmetProvider>
-        <Provider store={store}>
-          <SubstrateProvider>
-            <ThemeWrapper>
-              <SubstrateLoadingAndErrorHandlingWrapper>
-                <Subscriptions>
-                  <OverlayProvider>
-                    <App />
-                  </OverlayProvider>
-                </Subscriptions>
-              </SubstrateLoadingAndErrorHandlingWrapper>
-            </ThemeWrapper>
-          </SubstrateProvider>
-        </Provider>
-      </HelmetProvider>
-      <ReactQueryDevtools initialIsOpen={false} />
-    </QueryClientProvider>
-  </Router>,
+  <GeoblockingWrapper>
```

Comment:
App is wrapped in geoblocking component to make sure user is redirected as soon as possible.